src/main/groovy/jaci/openrio/gradle/ExternalLaunchTask.groovy

```diff
@@ -13,6 +13,7 @@ class ExternalLaunchTask extends DefaultTask {
     private def _withBuilderClosures = [] as List<Closure>
     def environment = [:] as Map<String, String>
     def persist = false
+    def headless = project.hasProperty('headless')
```

Comment:
This'll need to be checked in the task action, since otherwise if the headless property is set in the DSL it'll be too late